controllers/articles-controller.js

```diff
@@ -16,7 +28,21 @@ exports.getArticles = (req, res, next) => {
 };
 
 exports.getAllArticles = (req, res, next) => {
-  selectAllArticles()
+  let sortBy = "";
+  let order = "";
+  if (Object.keys(req.query).length === 0) {
+    sortBy = "created_at";
+    order = "DESC";
+  } else {
+    sortBy = req.query.sort_by;
+    order = req.query.order.toUpperCase();
```

Comment:
You could use default parameters in your function to handle this. In your model you could have `selectAllArticles(sortBy = 'created_at', order='DESC')` and that would omit the need for this if statement.

Reply:
<img width="288" alt="Screenshot 2024-08-27 at 14 46 10" src="https://github.com/user-attachments/assets/45cfdaae-d60b-479b-87d2-e75f7b18da00">